Explorer/Assets/DCL/Friends/UI/FriendPanel/Sections/Blocked/BlockedUserView.cs

```diff
@@ -19,7 +20,7 @@ public DateTime BlockedDate
             set
             {
                 blockedDate = value;
-                BlockedDateText.SetText($"{blockedDate:MM/dd}");
+                BlockedDateText.SetText(blockedDate.ToString("MMM dd", CultureInfo.InvariantCulture).ToUpper());
```

Comment:
ToString and ToUpper allocate a new string each time, there should be a way of improving this with a span considering that you should know the string length 🤔